recipe/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export ARCTICDB_USING_CONDA=1
+export CMAKE_BUILD_PARALLEL_LEVEL=1
```

Comment:
```suggestion
# Compiling ArcticDB with all cores might freeze machines due to swapping.
# We build with only 1 core to prevent these freezes from happening.
export CMAKE_BUILD_PARALLEL_LEVEL=1
```